x/concentrated-liquidity/position_test.go

```diff
@@ -191,17 +189,19 @@ func (s *KeeperTestSuite) TestInitOrUpdatePosition() {
 				// LastLiquidityUpdate should not have moved up since init upon error
 				clPool, err = s.App.ConcentratedLiquidityKeeper.GetPoolById(s.Ctx, clPool.GetId())
 				s.Require().NoError(err)
-				s.Require().Equal(defaultJoinTime, clPool.GetLastLiquidityUpdate())
+				s.Require().Equal(DefaultJoinTime, clPool.GetLastLiquidityUpdate())
 				return
 			}
 			s.Require().NoError(err)
 
 			// Get the tick info for poolId 1
 			positionInfo, err = s.App.ConcentratedLiquidityKeeper.GetPosition(s.Ctx, validPoolId, s.TestAccs[0], test.param.lowerTick, test.param.upperTick, test.param.joinTime, test.param.freezeDuration)
 			s.Require().NoError(err)
-			fmt.Println(positionInfo)
-			// Check that the initialized or updated position matches our expectation
-			s.Require().Equal(test.expectedLiquidity, positionInfo.Liquidity)
+
+			if !test.positionExists {
+				// Check that the initialized or updated position matches our expectation
+				s.Require().Equal(test.expectedLiquidity, positionInfo.Liquidity)
+			}
```

Comment:
I don't think this check should be here, it should be the same as before `s.Require().Equal(test.expectedLiquidity, positionInfo.Liquidity)` because this is a validation for when we are expecting `InitOrUpdatePosition` to be successful

Reply:
ah yes readded!